### PR TITLE
Update the first quick guide

### PR DESCRIPTION
**Pull request by rayzhou2017**

### CODE REVIEW

**Comment by rayzhou2017** [comment]
The devops is changed to the workspace level, so please update any guide related to this change. @FeynmanZhou

File: content/v2.1/en/quick-start/admin-quick-start.md [hunk 1]
```diff
@@ -7,42 +7,42 @@ description: ''
...
 ## Hands-on Lab
 
-The resources of KubeSphere has been divided into **three** levels which are Cluster, Worksapce, Project and DevOps Project.
+KubeSphere system is organized into **three** hierarchy levels which are Cluster, Workspace and Project. Here a project is a Kubernetes namespace.
 
-As it shown below, you can create multiple workspaces under a Kubernetes cluster, for each workspace you can also create multiple project and DevOps project.
+As shown below, you can create multiple workspaces within a Kubernetes cluster. Under each workspace you can also create multiple projects.
 
-For each level, there are multiple built-in roles by default, it allows you to customize role as well. This hierachy is appropriate for enterprise who have different teams or groups, as well as different roles within each team.
+For each level, there are multiple built-in roles. It allows you to create role with customized authorization as well. This hierarchy is appropriate for enterprise uaers who have different teams or groups, as well as different roles within each team.
```
Comment (FeynmanZhou) [inline content/v2.1/en/quick-start/admin-quick-start.md]: uaers  => users

**Comment by FeynmanZhou** [comment]
> The devops is changed to the workspace level, so please update any guide related to this change. @FeynmanZhou

@rayzhou2017 Are you mentioning [this step](https://kubesphere.com.cn/docs/v2.1/en/quick-start/admin-quick-start/#step-5-create-devops-project-optional)?  

image

**Comment by FeynmanZhou** [comment]
> The devops is changed to the workspace level, so please update any guide related to this change. @FeynmanZhou

@rayzhou2017  I don't understand `The devops is changed to the workspace level`, as I see DevOps project is still in workspace.

image

**Comment by rayzhou2017** [comment]
> > The devops is changed to the workspace level, so please update any guide related to this change. @FeynmanZhou
> 
> @rayzhou2017 I don't understand `The devops is changed to the workspace level`, as I see DevOps project is still in workspace.
> 
> image

Forget it . I misunderstood.

**Pull request merged**